client/src/components/SearchResultsOffset.js

```diff
@@ -0,0 +1,167 @@
+import React from 'react'
+import { Box, Button, Paragraph, TextInput } from 'grommet'
+import { FormPrevious, FormNext } from 'grommet-icons'
+import { isOnlyNumbers } from '../helpers/isOnlyNumbers'
+import { useSearchResources } from '../hooks/useSearchResources'
+
+export const SearchResultsOffset = () => {
+  const {
+    query: { limit, offset },
+    response: { count },
+    setOffset,
+    goToSearchResults
+  } = useSearchResources()
+
+  const [enteredPageNumber, setEnteredPageNumber] = React.useState('')
+  const [
+    enteredPageNumberInRange,
+    setEnteredPageNumberInRange
+  ] = React.useState(false)
+
+  const safeOffset = parseInt(offset, 10) || 0
+  const safeCount = parseInt(count, 10) || 0
+  const safeLimit = parseInt(limit, 10)
```

Comment:
It seems odd to me that safeLimit might be NaN. Perhaps `parsedLimit` might be better?